apps/ledger-live-mobile/ios/ledgerlivemobile.xcodeproj/project.pbxproj

```diff
@@ -803,7 +803,11 @@
 					"$(inherited)",
 					" ",
 				);
-				OTHER_LDFLAGS = "$(inherited)";
+				OTHER_LDFLAGS = (
```

Comment:
is this code in this file supposed to have changed?

Reply:
I removed the change, it comes from the latest Xcode version.